GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -424,6 +425,7 @@ protected override void OnFormClosing(FormClosingEventArgs e)
             }
 
             _splitterManager.SaveSplitters();
+            AppSettings.CommitDialogShowOnlyMyMessages = ShowOnlyMyMessagesToolStripMenuItem.Checked;
```

Comment:
This should be done in the `OnClick` handler (as for other settings, too).

Reply:
Maybe it's better in on CheckedChanged?